### ISSUE TITLE
See if we can do something smart to reduce texture sizes

### ISSUE THREAD

**Issue by gfodor** [MozillaReality/Spoke#241]
If you are bringing in props to a scene and scaling them down it's often the case that the textures could be quite big despite the objects being relatively small in the scene. It might be worth trying to keep the scene size under some amount and using texture resizing etc to reduce scene size. (We have a 100MB limit right now.)

**Comment by brianpeiris** [comment]
I think we can get away without this for V1?

**Comment by gfodor** [comment]
Yup I think so, I am not adding issues to the project if they are not for v1
